routes/demos.js

```diff
@@ -42,15 +45,23 @@ module.exports = function (express) {
           });
         },
 
-        // Copy requested demo to the project.
+        // Download requested demo to the project folder.
         (project, projectPath, callback) => {
-          // TODO: take the sample from external url.
-          const samplePath = path.join(process.cwd(), 'sample', project.type, project.name);
-          fse.copy(samplePath, projectPath, function (err) {
-            if (err) {
-              return callback(err);
+          const tauExamplesHost = config.get('TAUExamplesHost');
+          const samplePath = req.query.path;
+          if (!tauExamplesHost || !samplePath)
+            return callback(`Could not resolve path: ${tauExamplesHost}${samplePath}`);
+
+          const sampleUrl = new URL(samplePath, tauExamplesHost);
+          // Define number of unnecessary directories to be omitted while downloading.
+          const numDirsToCut = sampleUrl.pathname.match(/TAU\/examples\/mobile|werable\/UIComponents/g) ? 4 : 0;
+          exec(`wget --page-requisites --convert-links --no-host-directories --cut-dirs=${numDirsToCut} --directory-prefix ${projectPath} ${sampleUrl}`, (error, stdout, stderr) => {
+            if (error) {
+              // FIXME: background-image's url from tau.css references file that
+              // does not exist on server. Uncomment line below once it's fixed.
+              // return callback(error);
```

Comment:
let's put console.log here so we don't miss it in future

Reply:
Done.